cme/README.md

```diff
@@ -18,7 +18,7 @@ This CME module uses **lsassy** to remotely extract lsass password, and optionna
 * Reinstall CrackMapExec using python2.7 `python setup.py install`
 
 ```bash
-git clone git@github.com/Hackndo/lsassy.git
+git clone https://github.com/Hackndo/lsassy.git
```

Comment:
I undid the change, and put back git@github.com:Hackndo/lsassy.git
Your public key needs to be assigned to your github account :)

Reply:
You can update this with the two solutions :) git or ssh